src/_locales/fr/messages.json

```diff
@@ -120,24 +120,24 @@
     "hash": "a2d1d2fdc32ecb73ffdf0ab537e46f32"
   },
   "messageAutoSelectColorButton": {
-    "message": "Utilise des couleurs en contraste.",
+    "message": "Utilisez des couleurs contrastées.",
```

Comment:
Not sure about what is this should mean, so both "Utilise" and "Utilisez" (as well as "Utiliser") *could* be correct. Maybe @rugk can give a bit more of context ?

Reply:
I agree with "contrastées" 👍

Reply:
I also struggled to understand the sentence before I read it in the addon itself.
Go to addon preferences and select two same colors (front and background), you will see the error message :wink:

Reply:
> I agree with "contrastées" +1

:+1:

Reply:
> The text of a button that sets a new color with a sufficient contrast.

It#s a button, you can click on. So it should be imperative.

https://github.com/rugk/offline-qr-code/blob/a6eba310a10b2fb003a34d3938587b9ad5d273b1/src/_locales/en/messages.json#L103-L106